voronoi_game.py

```diff
@@ -107,10 +104,8 @@ def __init__(self, player_list, args):
 
         self.add_players(player_list)
 
-        self.play_game()
-        self.end_time = time.time()
```

Comment:
We run `play_game()` after initializing the object. Allows switching gui on/off.
`init()` should not automatically start the sim.